data/examples/declaration/warning/warning-single-line-out.hs

```diff
@@ -1,4 +1,5 @@
-{-# WARNING test "This is a warning" #-}
 {-# DEPRECATED test, foo "This is a deprecation" #-}
+
+{-# WARNING test "This is a warning" #-}
```

Comment:
What do you check with this change? If it's just the presence of newline after deprecation annotation, then wouldn't it be already printed even without warning annotation following it? (I.e. newline would be there between deprecation annotation and `test :: IO ()`.)

Reply:
I was testing the absence of newline after the `WARNING` annotation. After this PR it's only adjacent to the `test` function because of the [line 143 
 of Declaration.hs](https://github.com/tweag/ormolu/pull/263/files#diff-0ccd0f9b89649783f9d4f7e7a6dc9e5eR143)

Reply:
I think we can keep it "as is".